storage/innobase/log/log0recv.cc

```diff
@@ -4236,7 +4236,7 @@ dberr_t recv_recovery_from_checkpoint_start(log_t &log, lsn_t flush_lsn,
       ut_ad(0);
       recv_sys->found_corrupt_log = true;
       return DB_ERROR;
-}
+  }
```

Comment:
remove this

Reply:
If I remove clang-format complains. So let it be?

Reply:
sure, if it is by clang-format, please keep it